care/emr/models/__init__.py

```
@@ -1,12 +1,31 @@
+from .charge_item import *  # noqa F403
+from .charge_item_definition import *  # noqa F403
 from .condition import *  # noqa F403
-from .observation import *  # noqa F403
-from .questionnaire import *  # noqa F403
-from .valueset import *  # noqa F403
-from .scheduling import *  # noqa F403
+from .device import *  # noqa F403
+from .diagnostic_report import *  # noqa F403
 from .encounter import *  # noqa F403
-from .patient import *  # noqa F403
 from .file_upload import *  # noqa F403
```

Pylint (3.3.7)

[error] Cannot import 'file_upload' due to 'f-string: expecting '}' (care.emr.models.file_upload, line 37)'

(E0001)